.github/workflows/check-dependencies.yml

```diff
@@ -3,7 +3,6 @@ name: "3rd-party"
 on:
   push:
     branches:
-      - master
       - /^release-.*$/
   pull_request:
```

Comment:
Due to automatic assignment of base-ref/head-ref on pull_request, their absence is noticed when pushing to master.
[Doc link 👇](https://github.com/actions/dependency-review-action#:~:text=base%2Dref/,none)

![image](https://github.com/apache/incubator-hugegraph/assets/87920097/4a78b17a-85b1-42ce-824a-6150747adad3)